homeassistant/components/thethingsnetwork/__init__.py

```diff
@@ -1,30 +1,24 @@
 """Support for The Things network."""
 
+import logging
+
 import voluptuous as vol
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
-import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import issue_registry as ir
 from homeassistant.helpers.typing import ConfigType
 
-CONF_ACCESS_KEY = "access_key"
-CONF_APP_ID = "app_id"
-
-DATA_TTN = "data_thethingsnetwork"
-DOMAIN = "thethingsnetwork"
+from .const import CONF_APP_ID, CONF_HOSTNAME, DOMAIN, PLATFORMS, TTN_API_HOSTNAME
+from .coordinator import TTNCoordinator
 
-TTN_ACCESS_KEY = "ttn_access_key"
-TTN_APP_ID = "ttn_app_id"
-TTN_DATA_STORAGE_URL = (
-    "https://{app_id}.data.thethingsnetwork.org/{endpoint}/{device_id}"
-)
+_LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_APP_ID): cv.string,
-                vol.Required(CONF_ACCESS_KEY): cv.string,
-            }
```

Comment:
We should retain the old config, else the boot would fail

Reply:
Oh right, I missed that. Hmmmmm. I guess this works

Reply:
I asked around, and we should keep the old config schema, so it's still visibile what the config schema was.

Reply:
re-added